website/content/en/status/report-2023-04-2023-06/mfsbsd-building-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== Purpose of this project
+
+This project creates an additional target of the weekly snapshots of -current and -stable versions of mfsBSD images in the src/release makefile.
+Currently, only the release versions of mfsBSD images are produced.
+This means they tend to get out of sync with the tools in base.
+This project aims to address that problem.
+
+==== What is mfsBSD?
```

Comment:
I would move this section up: I think it makes more sense to first define mfsBSD, then talk about your mfsBSD project.